spec/unit/canonical_path_spec.rb

```diff
@@ -75,11 +75,11 @@
   end
 
   context 'with no tenant id' do
-    it 'should not be parseable' do
+    xit 'should not be parseable' do
```

Comment:
`xit` disables the test

Before, I assumed it's a nonsense to create a canonical path without specifying a tenant. But now, all the methods in the inventory api that accept the canonical path, don't read the tenant id from it. It can be there, but don't have to. This test is testing if the creation of the `CanonicalPath` object fails if the tenant id is not specified.

oh an example:

``` ruby
path = CanonicalPath.new(feed_id: feed_id, resource_type_id: hawk_escape_id('WildFly Server'))
@client.list_resources_for_type(path.to_s, fetch_properties: false)
```

There is no need to specify the tenant id, because it's derived from the credentials in the inventory (this may change with the tenant header)